torch/lib/THS/generic/THSTensor.c

```diff
@@ -560,6 +557,7 @@ void THSTensor_(free)(THSTensor *self)
     return;
   if(THAtomicDecrementRef(&self->refcount))
   {
+    THFree(self->size);
```

Comment:
Btw, I just noticed `size` was never freed in the previous implementation. Memory leak?